src/play-item.js

```diff
@@ -0,0 +1,42 @@
+import {setup} from './auto-advance.js';
+
+/**
+ * Removes all remote text tracks from a player.
+ *
+ * @param  {Player} player
+ */
+const clearTracks = (player) => {
+  let tracks = player.remoteTextTracks() || [];
+
```

Comment:
Not sure what you mean by "does `forEach` cache the array's value". I don't think it caches anything past the iteration.